nuxeo-wopi-web-ui/src/main/resources/web/nuxeo.war/ui/nuxeo-wopi/nuxeo-wopi-link.html

```diff
@@ -0,0 +1,105 @@
+<!--
+(C) Copyright 2018 Nuxeo SA (http://nuxeo.com/) and contributors.
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+​
+    http://www.apache.org/licenses/LICENSE-2.0
+​
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+Contributors:
+  Thomas Roger <troger@nuxeo.com>
+-->
+
+<!--
+`nuxeo-wopi-link`
+@group Nuxeo UI
+@element nuxeo-wopi-link
+-->
+<dom-module id="nuxeo-wopi-link">
+  <template>
+    <style include="iron-flex iron-flex-alignment">
+      :host {
+        display: inline-block;
+      }
+    </style>
+
+    <template is="dom-if" if="[[_isAvailable(blob)]]">
+      <a href="[[_wopiURL(document, blob, xpath)]]" tabindex="-1" target="_blank">
+        <paper-icon-button raised src="[[_wopiIcon(blob)]]" id="wopiLink"></paper-icon-button>
+      </a>
+      <paper-tooltip for="wopiLink">[[_wopiTooltip(blob)]]</paper-tooltip>
+    </template>
+
+  </template>
+
+  <script>
+    Polymer({
+      is: 'nuxeo-wopi-link',
+      behaviors: [Nuxeo.I18nBehavior],
+      properties: {
+        document: Object,
+        blob: Object,
+        xpath: String,
+      },
+      appsToSupportedExtensions: {
+        word: ['doc', 'docm', 'docx', 'dot', 'dotm', 'odt', 'rtf', 'wopitest'],
+        excel: ['csv', 'ods', 'xls', 'xlsb', 'xlsm', 'xlsx'],
+        powerpoint: ['odp', 'pot', 'potm', 'potx', 'pps', 'ppsm', 'ppsx', 'ppt', 'pptm', 'pptx'],
+      },
```

Comment:
we usually add empty lines between function def for sake of readibility

Reply:
see https://github.com/nuxeo/nuxeo-web-ui/blob/master/.eslintrc.json for lint rules